aries-site/src/examples/components/pagination/PaginationListExample.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Box, List, Menu } from 'grommet';
+import { More } from 'grommet-icons';
+
+const data = [];
+
+for (let i = 0; i < 95; i += 1) {
+  data.push({
+    entry: `entry-${i + 1}`,
+  });
+}
+
+export const PaginationListExample = () => (
+  <Box overflow="auto" pad="xsmall">
+    <List
+      data={data}
+      action={(item, index) => (
+        <Menu
+          key={index}
+          icon={<More />}
+          hoverIndicator
+          items={[{ label: 'See Details' }, { label: 'Delete' }]}
+        />
+      )}
+      step={7}
+      show={{ page: 7 }}
+      paginate={{
```

Comment:
Maybe future we can look at already having these styles in theme when paginate is being used in table or list seems like a lot of props are being passed

Reply:
Yes agreed. It's just tricky to pass in theme because if a user uses pagination outside of this prop, they might not want it to come with the border top, etc. We can figure out what would be best default as we start using it more.

Reply:
What is the 80% use case according to the designs and requirments?